### PR TITLE
Added Apriltag Object Recognition

### PR DESCRIPTION
**Pull request by wmcclinton**
This PR adds Apriltag based object recognition inside the grasp skill. It will chose an image based grasp location at the center of the first fiducial it finds.

### CODE REVIEW

File: predicators/spot_utils/spot_utils.py [hunk 1]
```diff
@@ -401,6 +402,24 @@ def arm_object_grasp(self) -> None:
         global g_image_click, g_image_display
         g_image_display = img
         cv2.imshow(image_title, g_image_display)
+
+        if CFG.spot_grasp_use_apriltag:
+            # Convert Image to grayscale
+            gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+
+            # Define the AprilTags detector options and then detect the tags.
+            print("[INFO] detecting AprilTags...")
```
Comment (tsilver-bdai) [inline predicators/spot_utils/spot_utils.py:411]: let's change these to `logging.info`

**Review by tsilver-bdai — APPROVED** [review]
LGTM! let's test it again before merging

**Review by nkumar-bdai — APPROVED** [review]
LGTM! Exciting stuff!

**Review by nkumar-bdai — APPROVED** [review]
LGTM! Exciting stuff!

**Pull request merged**